elkodon_bb/log/src/logger/log.rs

```diff
@@ -0,0 +1,30 @@
+use crate::LogLevel;
+
+pub struct Logger {
+    _priv: (),
+}
+
+impl Logger {
+    pub const fn new() -> Self {
+        Self { _priv: () }
+    }
+}
+
+impl crate::logger::Logger for Logger {
+    fn log(
+        &self,
+        log_level: crate::LogLevel,
+        origin: std::fmt::Arguments,
+        formatted_message: std::fmt::Arguments,
+    ) {
+        let origin = format!("{}", origin);
+        match log_level {
+            LogLevel::Trace => log::trace!(target: &origin, "{}", formatted_message),
```

Comment:
I haven't used feature flags by myself yet so I'm wondering how to user is informed that there are feature flags. I guess it will be automatically generated from rustdoc. Do you have more knowledge about this?

Reply:
No, not yet, and I am also not yet happy about the solution here. I just wanted to create this PR so that you know where the journey is going.

Reply:
I looked it up, and it seems some crates list the features at the end of the documentation in docs.rs. So I added them to the `lib.rs` file at the end.